src/main/java/it/gov/pagopa/pu/classification/enums/AssessmentStatus.java

```diff
@@ -3,5 +3,6 @@
 public enum AssessmentStatus {
```

Comment:
@domenicogi , @marcomatteuccieng
what's the meaning of `NEW` status? should it be changed with `ACTIVE`?
what's the difference between `CLOSED` and `CANCELLED`?

CC: @robertopasetti